<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLCDNumber" name="number_1"/>
         </item>
         <item>
          <widget class="QPushButton" name="start_1">
           <property name="text">
            <string>Iniciar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pause_1">
           <property name="text">
            <string>Pausa</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stop_1">
           <property name="text">
            <string>Detener</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLCDNumber" name="number_2"/>
         </item>
         <item>
          <widget class="QPushButton" name="start_2">
           <property name="text">
            <string>Iniciar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pause_2">
           <property name="text">
            <string>Pausa</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stop_2">
           <property name="text">
            <string>Detener</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLCDNumber" name="number_3"/>
         </item>
         <item>
          <widget class="QPushButton" name="start_3">
           <property name="text">
            <string>Iniciar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pause_3">
           <property name="text">
            <string>Pausa</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stop_3">
           <property name="text">
            <string>Detener</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
